zproject/settings.py

```diff
@@ -173,6 +173,7 @@ def get_secret(key):
                     'PERSONAL_ZMIRROR_SERVER': None,
                     'EXTRA_INSTALLED_APPS': [],
                     'DEFAULT_NEW_REALM_STREAMS': ["social", "general", "zulip"],
+                    'DEFAULT_NEW_REALM_STREAM_DESCRIPTIONS': ["For socializing", "For general stuff", "For Zulip discussion"],
```

Comment:
I think it's a little brittle to have parallel arrays here.  It's a little tricky, because we do need to continue to support `DEFAULT_NEW_REALM_STREAMS` for sites that have already configured their settings.  But I'd consider make `DEFAULT_NEW_REALM_STREAM_DESCRIPTIONS` be a dictionary, which is a little more verbose, but slightly less error prone.